app/Resources/views/blog/_comment_form.html.twig

```diff
@@ -20,6 +20,7 @@
             {{ form_label(form.content, 'label.content', { label_attr: { class: 'hidden' }}) }}
 
             {% if not form.vars.valid %}
+                {{ form_errors(form) }}
                 {{ form_errors(form.content) }}
```

Comment:
@xelaris, I would remove this string:
```twig
{{ form_errors(form.content) }}
```
Now it's not needed.
And I think you should remove the `if`  statement,  because the  `form_errors` doesn't produce any output when no errors occurred.